pkg/new-kernel/Dockerfile

```diff
@@ -272,13 +272,18 @@ RUN if [ "${EVE_TARGET_ARCH}" != riscv64 ]; then \
             SIG_KEY_SRCPREFIX=$(sed -n '/^MODULE_SIG_KEY_SRCPREFIX=/s///p' ${KERNEL_DEF_CONF} | tr -d '"') ; \
             SIG_KEY=$(sed -n '/^CONFIG_MODULE_SIG_KEY=/s///p' ${KERNEL_DEF_CONF} | tr -d '"') ; \
             scripts/sign-file "${SIG_HASH}" "/linux/${SIG_KEY_SRCPREFIX}${SIG_KEY}" /linux/certs/signing_key.x509 $(find /tmp/kernel-modules/lib/modules -name 8821cu.ko); \
+            if [ "${EVE_TARGET_ARCH}" = aarch64 ];then \
```

Comment:
we have zfs on riscv64 too, only 8821cu is not installed on riscv64.

Reply:
nice catch, that's need to be different for new-kernel, I will fix...